chore(APP-316): Update dependencies in package.json and pnpm-lock.yaml

tyhonchik opened the pull request:
# Description

- Bump versions for @reown/appkit, @reown/appkit-adapter-wagmi, @reown/walletkit, @sentry/core, @sentry/nextjs, wagmi, eslint, and vercel.
- Update package manager version to pnpm@10.24.0.
- Add package extensions for @solana/codecs-strings to have proper peer-deps.
- Update js-yaml and resolve dependencies in snapshots.

## Type of Change

<!--- Please delete the options that are not relevant. -->

- [ ] Major: Breaking change (change that would cause existing functionality to not work as expected)
- [ ] Minor: Feature (non-breaking change which adds new functionality)
- [x] Patch: Enhancement (non-breaking change to an existing feature)
- [ ] Patch: Bug fix (non-breaking change which fixes an issue)

## Developer Checklist:

- [ ] Manually smoke tested the functionality in a preview or locally
- [ ] Confirmed there are no new warnings or errors in the browser console
- [ ] (For User Stories only) Double-checked that all Acceptance Criteria are satisfied
- [ ] Confirmed there are no new warnings on automated tests
- [ ] Merged and published any dependent changes in downstream modules
- [ ] Selected the correct base branch
- [ ] Commented the code in hard-to-understand areas
- [ ] Followed the code style guidelines of this project
- [ ] Reviewed that the Files Changed in Github’s UI reflect my intended changes
- [ ] Confirmed the pipeline checks are not failing

## Review Checklist:

- [ ] (For User Stories only) Tested in a preview or locally that all Acceptance Criteria are satisfied
- [ ] Confirmed that changes follow the code style guidelines of this project

RakeshUP reviewed (changes requested):
Bang-up job @sepehr2github! Looks extraordinary

The NetworkErrorMenu doesn't show the "Switch Network" button for me since the provider connection URL has been changed from `metamask` to `eip-1193:` on this line https://github.com/aragon/app/blob/f/app-2251-wcv2/packages/web-app/src/containers/networkErrorMenu/index.tsx#L77

Apart from that everything looks good to me

tyhonchik committed:
chore(APP-316): Update dependencies in package.json and pnpm-lock.yaml

- Bump versions for @reown/appkit, @reown/appkit-adapter-wagmi, @reown/walletkit, @sentry/core, @sentry/nextjs, wagmi, eslint, and vercel.
- Update package manager version to pnpm@10.24.0.
- Add package extensions for @solana/codecs-strings to have proper peer-deps.
- Update js-yaml and resolve dependencies in snapshots.

tyhonchik commented:
we still need to wait cooldown period to update to the latest vercel and latest sentry with security issues
